src/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  flex-grow: 1;
+  justify-content: space-between;
+  padding: 0 50px;
+  min-width: 1024px;
+  max-width: 1200px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
if you want to remove default values like margin and padding remove them for ALL tags, use tag for this instead of a specific class